src/utils/colorManipulation.ts

```diff
@@ -42,26 +103,33 @@ export function computeDivisions(
 
   // TODO: this can be float, need to cast to int
   const canvasDivision: Dimensions = {
-    width: frame.width / horizontalDivisions,
-    height: frame.height / verticalDivisions,
+    width: Math.floor(frame.width / horizontalDivisions),
+    height: Math.floor(frame.height / verticalDivisions),
```

Comment:
nice, great quick fix. canvas performance can drop if we try to use floating points